src/Cosi-BLS/mvp-election-beacon.lisp

```diff
@@ -209,6 +209,14 @@ based on their relative stake"
    
 (defvar *mvp-election-period*  20) ;; seconds between election rounds
 
```

Comment:
NODE-STAKE slot in NODE structure no longer filled in because we get stakes from emotiq/config. So provide a way to find my own stake amount. Used only by diagnostic messaging.